cmd/clusterctl/client/cluster/ownergraph.go

```diff
@@ -39,11 +39,18 @@ type OwnerGraphNode struct {
 	Owners []metav1.OwnerReference
 }
 
+// GetOwnerGraphFilterFunction allows filtering the objects returned by GetOwnerGraph.
+// The function has to return true for objects which should be kept.
+// NOTE: this function signature is exposed to allow implementation of E2E tests; there is
+// no guarantee about the stability of this API. Using this test with providers may require
+// a custom implementation of this function, or the OwnerGraph it returns.
```

Comment:
This function doesn't return an OwnerGraph

Reply:
Removed that part.